src/go/rpk/pkg/cli/cmd/start.go

```diff
@@ -109,6 +111,30 @@ func NewStartCommand(fs afero.Fs, mgr config.Manager) *cobra.Command {
 				return err
 			}
 			conf.Redpanda.SeedServers = seedServers
+			advertisedKafka = stringOr(
+				advertisedKafka,
+				os.Getenv("REDPANDA_ADVERTISE_KAFKA_ADDRESS"),
```

Comment:
Advertise or Advertise**d**? IN the config file it has a d. There are a few of these; env var and flag.

Reply:
@rkruze pointed out that some Hashicorp tools, CockroachDB & Docker use "advertise", as opposed to Kafka's "advertised". I thought that was ok. Since the config fields themselves are called `advertised_*`, I named the env vars and the flags without the "ed" but kept the code consistent with the config field name.

Reply:
For Kafka it's something like: `KAFKA_ADVERTISED_LISTENERS=PLAINTEXT://internal-host:9092,OUTSIDE://external-host:9092` and has a different form. See #186

Reply:
I don't have strong opinions here. Just adding context.